### ISSUE TITLE
Different background color in markdown <code> blocks

### ISSUE THREAD

**Issue by josephrocca** [huggingface/huggingface_hub#767]
Here's what \`code\` blocks currently look like in README files:

![image](https://user-images.githubusercontent.com/1167575/158046530-d58382d2-c0d9-4ae5-9505-6c01dc84d81b.png)

I think something like this is much more readable:

![image](https://user-images.githubusercontent.com/1167575/158046507-507645ec-825b-483f-863a-0e4a13f2f67f.png)

In that example I've just added these 3 lines to the `.prose code` CSS block:

```
background-color: #e6e6e6;
border-radius: 2px;
padding: 0.05rem 0.2rem;
```

**Comment by julien-c** [comment]
WDYT @gary149?

**Comment by gary149** [comment]
I agree this is more readable 👍 . The background is a bit too dark but that's fine-tuning. @mishig25 can probably take a look at this (we need to be careful with the CSS cascading to not break anything).
